sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/implementation/CoreToCodegenBridgeUtils.java

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+package com.azure.messaging.servicebus.administration.implementation;
+
+import java.time.Duration;
+
+/**
+ * Utility class that handles functionality not yet available in azure-core that generated code requires.
+ */
+public final class CoreToCodegenBridgeUtils {
+
+    /**
+     * Converts a {@link Duration} to a string in ISO-8601 format with support for a day component.
+     *
+     * @param duration The {@link Duration} to convert.
+     * @return The {@link Duration} as a string in ISO-8601 format with support for a day component, or null if the
+     * provided {@link Duration} was null.
+     */
+    public static String durationToStringWithDays(Duration duration) {
```

Comment:
This class is only used in implementation/models. Can we put this as a package-private class rather than a public one?

Reply:
This was autogenerated. But this should be removable soon as this was a bridge for the generator while `azure-core` was adding a new feature.

The concern is that implementation can be accessed in Java 8 and if we change this it will break consumers, right? I do want to push back on that as we state implementation isn't public and therefore can break you if you use it.